tests/puppeteer/campaigns/functional/BO/13_shopParameters/03_productSettings/02_productPage/06_displayDiscountedPrice.js

```diff
@@ -0,0 +1,158 @@
+    catalogPriceRulesPage: new CatalogPriceRulesPage(page),
+    addCatalogPriceRulePage: new AddCatalogPriceRulePage(page),
+    homePage: new HomePage(page),
+    productPage: new ProductPage(page),
+  };
+};
+
+describe('Enable/Disable display discounted price', async () => {
+  // before and after functions
+  before(async function () {
+    browser = await helper.createBrowser();
+    page = await helper.newTab(browser);
+    this.pageObjects = await init();
+  });
+  after(async () => {
+    await helper.closeBrowser(browser);
+  });
+
+  // Login into BO and go to products page
+  loginCommon.loginBO();
+
+  it('should go to \'Catalog > Discounts\' page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToDiscountsPage', baseContext);
+    await this.pageObjects.boBasePage.goToSubMenu(
+      this.pageObjects.boBasePage.catalogParentLink,
+      this.pageObjects.boBasePage.discountsLink,
+    );
+    await this.pageObjects.boBasePage.closeSfToolBar();
+    const pageTitle = await this.pageObjects.cartRulesPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.cartRulesPage.pageTitle);
+  });
+
+  it('should go to \'Catalog Price Rules\' tab', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToCatalogPriceRulesTab', baseContext);
+    await this.pageObjects.cartRulesPage.goToCatalogPriceRulesTab();
+    const pageTitle = await this.pageObjects.catalogPriceRulesPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.catalogPriceRulesPage.pageTitle);
+  });
+
+  it('should create new catalog price rule', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'createCatalogPriceRule', baseContext);
+    await this.pageObjects.catalogPriceRulesPage.goToAddNewCatalogPriceRulePage();
+    const pageTitle = await this.pageObjects.addCatalogPriceRulePage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.addCatalogPriceRulePage.pageTitle);
+    const validationMessage = await this.pageObjects.addCatalogPriceRulePage.createEditCatalogPriceRule(priceRuleData);
+    await expect(validationMessage).to.contains(this.pageObjects.catalogPriceRulesPage.successfulCreationMessage);
+  });
+
+  it('should go to \'Shop parameters > Product Settings\' page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToProductSettingsPage', baseContext);
+    await this.pageObjects.boBasePage.goToSubMenu(
+      this.pageObjects.boBasePage.shopParametersParentLink,
+      this.pageObjects.boBasePage.productSettingsLink,
+    );
+    const pageTitle = await this.pageObjects.productSettingsPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.productSettingsPage.pageTitle);
+  });
+
+  const tests = [
+    {args: {action: 'disable', unitPrice: false, unitDiscount: true}},
+    {args: {action: 'enable', unitPrice: true, unitDiscount: false}},
```

Comment:
```suggestion
    {args: {action: 'disable', unitDiscount: true, textToCheck: 'Unit price'}},
    {args: {action: 'enable', unitPrice: true, textToCheck: 'Unit discount'}},
```

No need to add false values